docs/snippets/common/storybook-angular-inline-css-loader.js.mdx

```diff
@@ -0,0 +1,3 @@
+```js
+import '!style-loader!css-loader!./styles.css';
```

Comment:
I'm not sure if this should be an angular snippet or if the whole section should only be included for angular somehow (we don't have this mechanism yet)

Reply:
For now the best course of action would be to move the snippet to the angular folder and if the same issue rises in other frameworks we can move into common and rename it to something else.